api/apps/articles/model/save.coffee

```diff
@@ -290,6 +330,12 @@ typecastIds = (article) ->
     debug err if err
     cb()
 
+stripHtmlTags = (str) ->
```

Comment:
i imagine there is an implementation of this tucked into one of our dependencies but i couldn't find it.